Formula/jpeg-xl.rb

```diff
@@ -0,0 +1,63 @@
+class JpegXl < Formula
+  desc "New file format for still image compression"
+  homepage "https://jpeg.org/jpegxl/index.html"
+  url "https://gitlab.com/wg1/jpeg-xl/-/archive/v0.2/jpeg-xl-v0.2.tar.bz2"
+  sha256 "f0933c796f95ee905efa7a677367c0d57678b9587c2e967ea30d72e9405cca72"
+  license "Apache-2.0"
+
+  # The build dependencies are documented on the jpeg-xl osx-build page
+  # https://gitlab.com/wg1/jpeg-xl/-/blob/master/README.OSX.md
+  depends_on "clang-format" => :build
+  depends_on "cmake" => :build
+  depends_on "coreutils" => :build
+  depends_on "ninja" => :build
```

Comment:
"ninja" is only needed if you pass "-G Ninja" to cmake, which is the default if you use "./ci.sh release", so that's probably why it is there. I don't think you need that in this recipe.